docs/ipfs-cluster-guide.md

```diff
@@ -260,6 +261,12 @@ This involves:
 
 Errors in the first part of the process (before the entry is commited) will be returned to the user and the whole operation is aborted. Errors in the second part of the process will result in pins with an status of `PIN_ERROR`.
 
+Deciding where a CID will be pinned (which IPFS daemon will store it - receive the allocation) is a complex process. In order to decide, all available peers (those reporting valid/non-expired metrics) are sorted by the `allocator` component, in function of the value of their metrics (provided by the `informer` component). If a CID is already allocated to some peers (in the case of a re-pinning operation), those allocations are kept.
```

Comment:
I am unclear what you want to say by "in function of the value of their metrics".  Perhaps "as a function of the value" or "using a function of the value" or "in order of the value".

Reply:
`depending on the value of their metrics`